six/two/two.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include <algorithm>
 #include <sstream>
+#include <unistd.h>
```

Comment:
what's this for?

Reply:
unneeded now that the code is in `six/six`, good catch!